addons/payment_ogone/controllers/main.py

```diff
@@ -33,6 +33,40 @@ def ogone_s2s_create_json(self, **kwargs):
         new_id = request.env['payment.acquirer'].browse(int(kwargs.get('acquirer_id'))).s2s_process(kwargs)
         return new_id.id
 
+    @http.route(['/payment/ogone/s2s/create_json_3ds'], type='json', auth='public', csrf=False)
+    def ogone_s2s_create_json_3ds(self, **kwargs):
+        new_id = request.env['payment.acquirer'].browse(int(kwargs.get('acquirer_id'))).s2s_process(kwargs)
```

Comment:
And as discussed, if the implementation of the acquirer-specific `*_s2s_form_process()` methods is not consistent, do not hesitate to fix them (e.g. always return a record, not an ID)